packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.styles.ts

```diff
@@ -0,0 +1,42 @@
+import { getGlobalClassNames, getTheme } from '@uifabric/styling';
```

Comment:
Where is this component on the PR deploy? Or if it's not in the generic components list, where can I find its usage?

Reply:
This is not a generic component.
This component is used in SelectedPeopleList.

Reply:
What is SelectedPeopleList? Can you point me to a link of the docs for this component?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx
Above is the component where we are using the EditingItem line number 88.